Source/WebKit/Platform/spi/ios/UIKitSPI.h

```diff
@@ -1544,6 +1548,12 @@ typedef NS_ENUM(NSUInteger, _UIContextMenuLayout) {
 @property (nonatomic, setter=_setSuppressSoftwareKeyboard:) BOOL _suppressSoftwareKeyboard;
 @end
 
+#if HAVE(PASTEBOARD_DATA_OWNER) || HAVE(LINK_PREVIEW)
```

Comment:
It feels like the proper condition should only be `HAVE(PASTEBOARD_DATA_OWNER)`, but _performAsDataOwner:block: gets called from: https://github.com/WebKit/WebKit/blob/da89ad902d18bbc6f8d03b805b8eaf859497fdf3/Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm#L11170
which is inside of a `ENABLE(IMAGE_ANALYSIS_ENHANCEMENTS)` condition. Above, we declare `_UIDataOwner` inside a `HAVE(LINK_PREVIEW)` condition.

Ideas for improving:

- We could move HAVE_PASTEBOARD_DATA_OWNER out of AdditionalFeatureDefines.h, and use it at the call site above
- We could remove the conditions are _UIDataOwner in this file, and declare these SPI more broadly

Reply:
After talking with @whsieh, I've done some cleanup around HAVE_PASTEBOARD_DATA_OWNER and split it out to #3726.